plexapi/base.py

```diff
@@ -144,7 +144,8 @@ def fetchItems(self, ekey, cls=None, **kwargs):
             and attrs. See :func:`~plexapi.base.PlexObject.fetchItem` for more details
             on how this is used.
         """
-        data = self._server.query(ekey)
+        timeout = kwargs.pop('timeout', None)
```

Comment:
This should probably be added as a real keyword arg timeout=None in the function definition and not pulled out of kwargs.